ddtrace/internal/utils/cache.py

```diff
@@ -69,7 +69,7 @@ def cached_wrapper(f):
         cache = LFUCache(maxsize)
 
         def cached_f(key):
-            # type: (T) -> S
+            # type: (Callable[[T], S]) -> S
```

Comment:
I think this is wrong: the `key` argument should be of type `T`. Overall, `cached_f` is of type `Callable[[T], S]`.